internal/provisioner/model/model.go

```diff
@@ -24,9 +24,13 @@ import (
 )
 
 const (
-	// OwningGatewayNameLabel is the owner reference label used for a Contour
-	// created by the gateway provisioner. The value should be the name of the Gateway.
-	OwningGatewayNameLabel = "projectcontour.io/owning-gateway-name"
+	// ContourOwningGatewayNameLabel is the Contour-defined owner reference label applied
+	// to generated resources. The value should be the name of the Gateway.
+	ContourOwningGatewayNameLabel = "projectcontour.io/owning-gateway-name"
```

Comment:
We can probably deprecate/remove this one eventually since it serves the same purpose, just need to be careful about compatibility